multi_label_trainer_tfidf.ipynb

```diff
@@ -4,10 +4,9 @@
   "metadata": {
```

Comment:
<pre class="ql-syntax" spellcheck="false">train_dataset = make_dataset(train_df, is_train=True)
validation_dataset = make_dataset(val_df, is_train=False)
test_dataset = make_dataset(test_df, is_train=False)
</pre><p> </p><p>I think keyword arguments make the code more readable.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/soumik12345/multi-label-text-classification/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>